runtime/doc/treesitter.txt

```diff
@@ -126,6 +126,11 @@ TSNode:named_child({index})                              *TSNode:named_child()*
 TSNode:child_containing_descendant({descendant})  *TSNode:child_containing_descendant()*
     Get the node's child that contains {descendant}.
 
+    Note: Deprecated, prefer |TSNode:child_with_descendant()|
+
+TSNode:child_with_descendant({descendant})  *TSNode:child_with_descendant()*
+    Get the node's child that contains {descendant} (includes {descendant}).
```

Comment:
I tried to keep the description here similar to the previous func (for simplicity) but it could probably be more descriptive, either now or later

Reply:
Some examples would help.

Reply:
@lewis6991 I am pretty bad at writing good documentation, let me know your thoughts on the example I wrote